src/Http/Models/ShippingZone.php

```diff
@@ -3,56 +3,44 @@
 namespace Jonassiewertsen\StatamicButik\Http\Models;
 
 use PHPUnit\Framework\Constraint\Count;
+use Symfony\Component\Intl\Countries;
 
 class ShippingZone extends ButikModel
 {
     protected $table = 'butik_shipping_zones';
 
     protected $guarded = [];
 
-    /**
-     * A shipping zone belongs to a shippin profile
-     */
-    public function profile()
-    {
-        return $this->belongsTo(ShippingProfile::class, 'shipping_profile_slug');
-    }
+    protected $casts = [
+        'countries' => 'collection'
+    ];
 
-    /**
-     * A shipping zone belongs to many countries
-     */
-    public function countries()
-    {
-        return $this->belongsToMany(
-            Country::class,
-            'butik_country_shipping_zone',
-            'shipping_zone_id',
-            'country_slug',
-        );
-    }
+    protected $hidden = [
+        'created_at',
+        'updated_at'
+    ];
 
-    /**
-     * Add a country to a shipping zone
-     */
-    public function addCountry($country)
+    protected $appends = ['countries_display'];
+
+    public function getCountriesDisplayAttribute()
     {
-        $this->countries()->attach($country);
+        return $this->countries->map(function($countryCode) {
```

Comment:
We do need to prevent that the map methods will be called on an empty countries collection. Maybe something like this?

```

if (! $this->countries) {
            return null;
        }
```

Reply:
Empty countries attribute will result in an empty collection and `map()` call won't be executed.
I've changed the method to use `Shipping\Country` helper instead of the `Intl\Countries` method to have a single point where the library is used.